horace_core/admin/get_syms.m

```diff
@@ -1,4 +1,4 @@
-function [info, ops] = get_syms(arg)
+function [info, sym] = get_syms(arg)
```

Comment:
please add comments for meaning of input and output arguments

I cannot see where sym is set

Reply:
The [python code](https://github.com/pace-neutrons/Horace/blob/259dba6ae7525c8e93464e5fff988040eea484e5/horace_core/admin/get_syms.py#L892) returns a string in the second return value which begins `sym = {` - this is evaluated by `evalc` into the function's workspace and returned directly to the user as `sym`.